File: BricksGDX/core/src/com/salvador/bricks/game_objects/SoundManager.java
package com.salvador.bricks.game_objects;

import static com.salvador.bricks.game_objects.ResourceManager.getMusic;
import static com.salvador.bricks.game_objects.ResourceManager.getSound;

import com.badlogic.gdx.audio.Music;
import com.badlogic.gdx.audio.Sound;

public class SoundManager {

    public static Music musicBackground;
    public static Sound brickSound;

    public static void loadSounds() {
        musicBackground = getMusic("music.mp3");
        brickSound = getSound("sound.ogg");
    }

    public static void playMusicBackground() {
        musicBackground.setLooping(true);
        musicBackground.play();
    }

    public static void stopMusicBackground() {
        musicBackground.dispose();
        musicBackground.stop();
    }

    public static void playBrickSound() {
        brickSound.play();
    }
}

File: BricksGDX/core/src/com/salvador/bricks/game_objects/SoundManager.kt
package com.salvador.bricks.game_objects

import com.badlogic.gdx.audio.Music
import com.badlogic.gdx.audio.Sound

object SoundManager {

    lateinit var musicBackground: Music
    lateinit var brickSound: Sound

    @JvmStatic
    fun loadSounds() {
        musicBackground = ResourceManager.getMusic("music.mp3")
        brickSound = ResourceManager.getSound("sound.ogg")
    }

    @JvmStatic
    fun playMusicBackground() {
        musicBackground.isLooping = true
        musicBackground.play()
    }

    @JvmStatic
    fun stopMusicBackground() {
        musicBackground.dispose()
        musicBackground.stop()
    }

    @JvmStatic
    fun playBrickSound() {
        brickSound.play()
    }
}
